### ISSUE TITLE
Accessibility for the webpage

### ISSUE THREAD

**Issue by sxd** [cloudnative-pg/cloudnative-pg.github.io#389]
During the evaluation of the OpenSSF best practices I find out that our site has some issue with accessibility using the following link https://wave.webaim.org/report#/https://cloudnative-pg.io we should try to address this issue, since this could affect a lot of people.

**Comment by yuvraj-kolkar17** [comment]
Hi @sxd,
I'd like to work on fixing the accessibility issues identified in this issue. 
I plan to:
1. Run the WAVE accessibility evaluation on cloudnative-pg.io to identify all specific issues
2. Fix common accessibility problems including:
   - Missing alt text on images
   - Color contrast issues
   - Form label associations
   - Heading hierarchy
   - Any other issues found in the WAVE report
3. Update the Hugo templates and CSS as needed
4. Test the changes locally before submitting a PR
 I'd be happy to take ownership and submit a pull request with the fixes.
Thanks!
